contrib/lkt/extensions/src/liblktlang-implementation-extensions.adb

```diff
@@ -36,7 +36,12 @@ package body Liblktlang.Implementation.Extensions is
      "@builtin struct Char {}" & ASCII.LF &
      "@builtin struct String " &
      "implements Sized, Indexable[Char], Iterator[Char] {}" & ASCII.LF &
+     "@builtin class LexicalEnv {" & ASCII.LF &
```

Comment:
You have a regression in the testsuite due to the fact that entities have moved. Run the testsuite with the `-r` option to change the test baseline to the test's actual output.

`./manage.py test <test_name> -r`

Reply:
Thanks for the hint!